policy-controller/src/main.rs

```diff
@@ -159,3 +177,67 @@ async fn sigterm() {
         _ => future::pending().await,
     };
 }
+
+fn log_init(LogLevel(level): LogLevel, format: LogFormat) -> Result<()> {
+    let (filter, _level) = reload::Layer::new(EnvFilter::new(level));
```

Comment:
we're not actually using the reload stuff yet, so should we maybe just not add it until we actually add reloading support?